src/styles/main.css

```diff
@@ -0,0 +1,233 @@
+
+html {
+  font-family: "Arial", "Helvetica", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 35px auto;
+  width: 720px;
+  padding: 10px;
+  gap: 1px;
+}
+
+.calendar__day {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100px;
+  width: 100px;
+  border: 1px solid #000;
+  background-color: #eee;
+  transition: transform 0.5s;
+}
+
+.calendar__day:hover {
+  background-color: #ffc0cb;
+  cursor: pointer;
+  transform: translateY(-20px);
+}
+
+.calendar__day:nth-child(1)::before {
+  content: "1";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(2)::before {
+  content: "2";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(3)::before {
+  content: "3";
+  font-size: 30px;
+}
+
+.calendar__day:nth-child(4)::before {
+  content: "4";
+  font-size: 30px;
+}
```

Comment:
Don't hardcode it. Consider using for loop instead
https://sass-lang.com/documentation/at-rules/control/for